### PR TITLE
Improve docker-compose

### PR DESCRIPTION
**Pull request by blazarecki**

### CODE REVIEW

**Comment by ogizanagi** [comment]
What is motivating those changes ? Can you explain what are the goals this PR target ?

File: features/bootstrap/FeatureContext.php [hunk 1]
```diff
@@ -46,7 +44,7 @@ public static function prepare(BeforeSuiteScope $event)
             $process->run();
 
             if (!$process->isSuccessful()) {
-                throw new \Exception($process->getErrorOutput());
+                throw new \Exception($process->getOutput().$process->getErrorOutput());
```
Comment (ogizanagi) [inline features/bootstrap/FeatureContext.php]: Actually, I'm not that familiar with the `symfony/process` component, but what about using a `ProcessFailedException` ? It seems to achieve what you intent to do, in a more complete way :smile:

Comment (blazarecki) [inline features/bootstrap/FeatureContext.php]: I was not aware of this. Thanks for the tips.

Fixed.

**Comment by blazarecki** [comment]
I had issues with docker with my testing environment. So this PR fix it. And I replace `nginx` and `php-fpm` with `php:apache` in order to have only one container for the web. Each container has it's own responsibility.

**Comment by ogizanagi** [comment]
> Each container has it's own responsibility.

It was already the case before, that's why I asked.

I don't know which issues you encountered in your testing environment, but tests should not start the web server right ? So I don't see how those changes are related.

Anyway, if there is no issue for your PR, you should at least expose what are your motivations, otherwise, we'll only be able to state if your PR seems good & coherent technically, but not if it is for the project nor if it solves the issues you target.

**Comment by blazarecki** [comment]
> Each container has it's own responsibility.
> > It was already the case before, that's why I asked.

Not really, together `web` (nginx) and `fpm` (php:fpm) were a php web server by being coupled. 

Now I only have one container to do this. This also allow me to not use entrypoint which make difficult to work with (In my opinion).

**Comment by blazarecki** [comment]
Merged via https://github.com/poledev/benjamin/pull/25

**Pull request closed without merging**